src/materials/czm/GBCavitation/ShamNeedlemanEquation.C

```diff
@@ -0,0 +1,823 @@
+#pragma once
```

Comment:
Please add a glossary mapping the short variable names to what they are in the header (along with comments on what each function is).